src/index.css

```diff
@@ -1,10 +1,8 @@
-body {
-  margin: 0;
-  font-family: "Roboto", -apple-system, BlinkMacSystemFont, "Segoe UI", "Oxygen", "Ubuntu", "Cantarell", "Fira Sans", "Droid Sans", "Helvetica Neue", sans-serif;
-  -webkit-font-smoothing: antialiased;
-  -moz-osx-font-smoothing: grayscale;
-}
+@import url('https://fonts.googleapis.com/css2?family=Lato:wght@300;400;700&display=swap');
 
-code {
-  font-family: "Courier New", source-code-pro, Menlo, Monaco, Consolas, monospace;
+body,
+html {
+  font-family: 'lato', sans-serif !important;
```

Comment:
As requested by the previous reviewer, kindly follow the design guidelines by using the right font-family `( gill sans, Lato )`. `gill sans` should be for the body and title font while as `lato` will be a control font.

Reply:
Hi, @muneebulrehman 
Thanks for the review.
I will be happy if you can point me to where I can get the gill sans fonts.
I could not find it so have to use lato since that is part of google font which is free